about.html

```diff
@@ -12,62 +13,137 @@
-        
-      <footer>
-        <nav>
-          <a href="index.html">Home</a>
-          <a href="#">About</a>
-          <a href="#">Products</a>
-          <a href="#">Blog</a>
-          <a href="#">Contact</a>
-        </nav>
-      </footer>
-    </div><!-- container -->        
+  <div class="container about-page">
+    <header>
+      <nav>
+        <img src="./img/lambda-black.png" alt="lambda-black.png">
+        <a href="./index.html">Home</a>
+        <a href="about.html">About</a>
+        <a href="#">Products</a>
+        <a href="#">Blog</a>
+        <a href="#">Contact</a>
+      </nav>
+
+    </header>
+    <div class="banner">
+      <img src="./img/jumbo-about.png" alt="jumbo banner about ">
+    </div>
+    <div class="about">
+
+      <h2>
+        About
+      </h2>
+      <p>
+        Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to
+        corporate
+        strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic
+        world
+        view of disruptive innovation via workplace diversity and empowerment.
+      </p>
+    </div>
+
+
+    <section class="about-content">
+      <div class="content-container">
+        <img src="img/about-plan.png" alt="strategy">
+
+        <h2>
+          Strategy
+        </h2>
+
+        <p>
+          Iterative approaches to corporate strategy foster collaborative thinking to further the overall value
+          proposition.
+        </p>
+
+        <div class="learn-more">
+          <a href="#">Learn More</a>
+        </div>
+
+      </div>
+
+
+      <div class="content-container">
+        <img src="img/about-working.png" alt="strategy">
+
+        <h2>
+          How We Work
+        </h2>
+
+        <p>
+          Iterative approaches to corporate strategy foster collaborative thinking to further the overall value
+          proposition.
+        </p>
+
+        <div class="learn-more">
+          <a href="#">Learn More</a>
+        </div>
+      </div>
+      <br />
+      <br />
+      <div class="content-container">
+        <img src="img/about-office.png" alt="strategy">
+
+        <h2>
+          Places We Work
+        </h2>
+
+        <p>
+          Iterative approaches to corporate strategy foster collaborative thinking to further the overall value
+          proposition.
+        </p>
+
+        <div class="learn-more">
+          <a href="#">Learn More</a>
+        </div>
+      </div>
+
+      <div class="content-container">
+        <img src="img/about-meeting.png" alt="strategy">
+
+        <h2>
+          Collaboration
+        </h2>
+
+        <p>
+          Iterative approaches to corporate strategy foster collaborative thinking to further the overall value
+          proposition.
+        </p>
+
+        <div class="learn-more">
+          <a href="#">Learn More</a>
+        </div>
+      </div>
+
+    </section>
```

Comment:
You could use prettier formatter VS code extensions to make your codes cleaner and properly formatted!